packages/design-system/src/components/list-position-indicator.tsx

```diff
@@ -16,6 +16,7 @@ const OVERLAP = 2;
 
 const Container = styled(Box, {
   position: "absolute",
+  zIndex: 1,
```

Comment:
would isolation: isolate work here?

Reply:
I don't think so, I just need to get it higher than outline